### PR TITLE
improvements to UI and readme

### PR DESCRIPTION
**Pull request by elof**
Please confirm I didn't break anything, merge this to master, and update the live demo app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,11 +13,13 @@ The federation url has to be provided in `Config.js` file. The user will then be
 
 ```js
 const Config = {
-    global: "gdn1.prod.macrometa.io",
-    Toronto: "gdn1-tor1.prod.macrometa.io",
-    London: "gdn1-lon1.prod.macrometa.io",
-    Bengaluru: "gdn1-blr1.prod.macrometa.io",
-    Singapore: "gdn1-sgp1.prod.macrometa.io"
+  global: "gdn.paas.prod.macrometa.io",
```
Comment (dwivedianurag) [inline README.md]: I think we need to change this to gdn.paas.macrometa.io

Comment (elof) [reply README.md]: yes, you're right. Please update it

**Review by dwivedianurag — CHANGES_REQUESTED** [review]
Please do the needful

File: README.md (hunk 1 above)
Comment (dwivedianurag) [reply README.md]: ok I have made the change

**Review by dwivedianurag — APPROVED** [review]
ok

**Pull request merged**